src/Services/Azure/Kusto/KustoClient.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System.Text.Json.Nodes;
+using Azure.Core;
+
+namespace AzureMcp.Services.Azure.Kusto;
+
+public class KustoClient
+{
+    #region Private Members
+    private readonly string _clusterUri;
+    private readonly IHttpClientFactory _httpClientFactory;
+    private readonly TokenCredential _tokenCredential;
+    private readonly static string s_application = "AzureMCP";
+    private readonly static string s_clientRequestIdPrefix = "AzMcp";
+    private readonly static string s_default_scope = "https://kusto.dev.kusto.windows.net" + "/.default";
```

Comment:
Do we want to support non-public cloud? Or all clouds use same scope string?

Reply:
@xiangyan99 -  an offline code referenced by the Kusto team indicates that this scope can be derived from information available in Azure.ResourceManager.Kusto.KustoClusterData object, which can be obtained from the subscription client. I don’t see the scope in this object but very likely it can be derived from other data present. For instance, it contains a management endpoint URI that varies across different Azure environments, there can be a mapping table from management Uri to scope Uri

I'm creating an issue for Kusto team to followup and add this functionality.